pkgs/top-level/release.nix

```diff
@@ -72,6 +72,9 @@ let
               jobs.wireshark.x86_64-darwin
               jobs.transmission-gtk.x86_64-darwin
 
+              # Musl
+              jobs.pkgsMusl.stdenv.x86_64-linux
```

Comment:
two things:

- this is in the Darwin-tested jobset, probably should be the nixos one.
- I don't think pkgsMusl is in jobs at all right now, you will need to add it like we do in haskell (https://github.com/NixOS/nixpkgs/pull/89425/files#diff-3f624960077266649716a0472a723f24L181)

Reply:
but after doing (2), it will be in the jobset without being an actual blocker